content/glossary.md

```diff
@@ -33,4 +66,16 @@ draft: true
 
 **Timeboard**
 
-**Trace**
\ No newline at end of file
+**Tracer**
+
+Datadog supports distributed tracing via the APM Integration. This allows you to instrument your existing code and see requests as they pass through all your systems. It works across different languages, databases, and RPC frameworks. Many languages are [supported natively][6]; however, developers can add their own language support by creating a custom shipper.
```

Comment:
worth pointing to how to do the custom shipper???

Reply:
See the dedicated documentation for more details.